README.md

```diff
@@ -1,2 +1,11 @@
-# time-traveler-boutique
+# Time Traveler Boutique
 An example repo to demonstrate various concepts for a LaunchCode WebDev class I'm teaching
+
+<img src="https://github.com/mcdev8/time-traveler-boutique/assets/71328737/92b0dab4-1d47-4a23-b4fa-a376f0827392" width="300" height="300">
+
+## Features (MVP):
+1. Timetravelers can create an account and login.
+2. Timetravelers can purchase items.
+3. Timetravelers can review items, edit and/or delete their reviews.
+4. Timetravelers can view a summary inventory and sales.
+6. (stretch goal) The website must allow for multiple roles with some view(s) restricted by role.
```

Comment:
These are great. What if we moved them to GitHub's project management tools or a similar tool like Trello?